.github/pull_request_template.md

```diff
@@ -6,3 +6,18 @@
 ## Solution
 
 - Describe the solution used to achieve the objective above.
+
+---
+
+## \[ Optional \] Changelog
```

Comment:
@alice-i-cecile @IceSentry So should we move the changelog and migration guide to the top or keep it at the bottom?

Reply:
Personally, I prefer both at the bottom, but I'm not opposed to either approach.

Reply:
I think we should probably do both at the bottom for consistency then.

Reply:
Yeah, I think keeping them at the bottom makes sense.

> It's nice to get a quick overview, but a changelog without the full context of the objective isn't necessarily better.

I think this is a good point. It's good at top for understanding a PR at a glance, but might be more confusing to an author conceptually (i.e. it sorta disrupts their mental flow).

Reply:
I'll leave it at the bottom then, unless anyone else has objections